kolibri/core/assets/src/views/nav-bar/index.vue

```diff
@@ -50,7 +50,7 @@
 
 <script>
 
-  const values = require('lodash.values');
+  const values = require('lodash/values');
```

Comment:
Yeah, lodash and all its is is there, it's just not resolving. I'll take another look. I prefer this approach though